packages/features/bookings/lib/getUserBooking.ts

```diff
@@ -29,6 +29,8 @@ const getUserBooking = async (uid: string) => {
           email: true,
           username: true,
           timeZone: true,
+          avatar: true,
```

Comment:
fetching avatar have cause some issues in the past because the size of payload becomes too big https://github.com/calcom/cal.com/pull/4846

Reply:
Yeah we are using avatarUrl only anyways forgot to remove avatar.